gf-webapp-admin/src/routes.ts

```diff
@@ -20,8 +19,4 @@ export const privateRoutes:IRoute[] = [
     path: MAIN_PAGE_ROUTE,
     component: MainPage,
   },
-  {
-    path: ROOMS_PAGE_ROUTE,
-    component: RoomPage,
-  },
 ];
```

Comment:
Add comments to clarify context. Hard to understand why that need

Reply:
Comments added